### PR TITLE
Prevent chart being too tall and going offscreen

### PR DESCRIPTION
**Pull request by adil192**
The chart is currently too tall on some screens (like mine) so I've set a max size* on its parent as suggested by the [Chart.js documentation](https://www.chartjs.org/docs/latest/configuration/responsive.html).
(*The actual property I've changed is max-width because the height is automatically scaled based on the parent's width)

### CODE REVIEW

**Comment by adil192** [comment]
@klausenbusk Is there anything preventing this being merged?

File: web/index.html [hunk 1]
```diff
@@ -35,7 +35,11 @@
 		</select>
 		<p style="display:inline" id="basic-stats"></p>
 
-		<canvas id="chart"></canvas>
+		<!-- max-width is ~2:1 (chart aspect ratio) to height, so canvas doesn't get too tall -->
+		<div style="max-width: 190vh; margin: 0 auto;">
```
Comment (klausenbusk) [inline web/index.html:39]: What is the rationale for going with `190` and not `200`?

Comment (adil192) [reply web/index.html:39]: 200 would fill up the entire screen with the chart. I made it smaller so we could see the other interface elements as well

**Comment by adil192** [comment]
Any updates?

**Comment by klausenbusk** [comment]
Any chance you could rebase this?

**Comment by adil192** [comment]
No problem!

**Comment by klausenbusk** [comment]
Sorry for this taking so long :)

**Comment by adil192** [comment]
No worries :)

**Comment by adil192** [comment]
@klausenbusk I'm making some UI improvements following this PR. If you approve of my changes, would you prefer
- a slew of pull requests for each change
- one single pull request with everything
- making me a contributor so I can commit to this repo without PRs

My initial changes look like this:
| Stock | Forked (light) | Forked (dark) |
| --- | --- | --- |
|  |  |

**Pull request merged**